azavea/climate-change-lab
# Split out some models and services into shared repository

## Overview

Split out some models and services into [shared repository](https://github.com/azavea/climate-change-components).

A future commit will also move over some components, with an end goal of
being able to share the chart components between two or more Angular applications.


### Demo

N/A. Everything should continue to work as before.


### Notes

Don't use `npm link` to develop in conjunction with changes in `climate-change-components` :disappointed: . See https://github.com/angular/angular-cli/issues/3854.


## Testing Instructions

 * Clone `azavea/climate-change-components` (`hub clone -p azavea/climate-change-components`)
 * Checkout https://github.com/azavea/climate-change-components/pull/3 (`git checkout feature/mvm/init`)
 * Install dependencies for `azavea/climate-change-components` (`yarn`)
 * Build and package the library (`yarn run build:library && npm pack`)
 * In this project, install the package library `cd ../climate-change-lab; npm install ../climate-change-components/climate-change-components-0.0.0.tgz`
 * Running the application should work without any errors (`yarn run serve`)

## Checklist

Closes #282
